### PR TITLE
Feat: Create Design Explore Section

### PR DESCRIPTION
**Pull request by cp-priyanka-g**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [x] Add MR description if applicable
- [x] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [x] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [x] Remove unused or commented code if not required
- [x] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [ ] In progress MR should be marked as draft
- [x] Make MR mark as ready before submitting it for review
- [x] Add images, videos, or gifs of UI 
- [x] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**
- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook  (chrome, firefox, safari)
- [ ] iPhone(safari)
- [x] Available android devices  (chrome, firefox)
- [x] iPad(safari)
- [x] Tablet (chrome, firefox)

**Note:**
**media query changes** 
1.Change in landing md for mobile (fix title accordingly ) , lg for desktop
2. Github Contribution.
3. Favourites.
4. Animated Creation
5. Reduce image height in unit test section.
6. Fix spacing of navigation in Weekly -mobile devices.
7. Devops section : fix spacing for absolute element.
8. replace checkbox color in apply jobs page.

**Description:**
Create design explore section.

**Video:**

https://github.com/canopas/canopas-website/assets/98794636/c5854ec2-d15a-44ed-8f1e-025c50059f6a

**Desktop performance:**
Screenshot from 2023-06-16 11-59-06

**Desktop performance:**
Screenshot from 2023-06-16 11-58-05

**Mobile Image:**
Screenshot from 2023-06-16 11-59-43
**Desktop Image:**
image

### CODE REVIEW

File: vue-frontend/src/components/contributions/DesignExplore.vue [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+          <swiper-slide v-for="(post, index) in data.slice(0, 5)" :key="index">
+            <div
+              class="tw-cursor-pointer image-content"
+              @click="openBlog(post.link)"
+            >
+              <img
+                :src="post.images[0]"
+                :srcset="`${post.images[0]} 400w,${post.images[1]} 800w`"
+                class="tw-w-fit tw-h-fit tw-object-cover tw-drop-shadow-md tw-rounded-lg"
+                alt="UnitTestImage"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/contributions/DesignExplore.vue:78]: Give relative names

File: vue-frontend/src/components/contributions/DesignExplore.vue [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+import { Swiper, SwiperSlide } from "swiper/vue";
+SwiperCore.use([FreeMode, Pagination, Autoplay]);
+
+export default {
+  data() {
+    return {
+      activeIndex: 0,
+      gridAnimation: "animation-paused",
+      gridAnimationReverse: "animation-running",
+      pausedId: 0,
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/contributions/DesignExplore.vue:164]: Hover is not working

File: vue-frontend/src/components/contributions/DesignExplore.vue [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+          <swiper-slide v-for="(post, index) in data.slice(5, 10)" :key="index">
+            <div
+              class="tw-cursor-pointer image2-content"
+              @click="openBlog(post.link)"
+            >
+              <img
+                :src="post.images[0]"
+                :srcset="`${post.images[0]} 400w,${post.images[1]} 800w`"
+                class="tw-w-fit tw-h-fit tw-object-cover tw-drop-shadow-md tw-rounded-lg"
+                alt="UnitTestImage"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/contributions/DesignExplore.vue:110]: Same as above comment

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/contributions/DesignExplore.vue (hunk 2 above)
Comment (cp-priyanka-g) [reply vue-frontend/src/components/contributions/DesignExplore.vue:164]: [screen-capture.webm](https://github.com/canopas/canopas-website/assets/98794636/288fe458-6806-4c90-b5af-064bd3b65c3b)

Comment (cp-priyanka-g) [reply vue-frontend/src/components/contributions/DesignExplore.vue:164]: its workinig for me.

Comment (cp-sumi-k) [reply vue-frontend/src/components/contributions/DesignExplore.vue:164]: What about desktop devices?

Comment (cp-priyanka-g) [reply vue-frontend/src/components/contributions/DesignExplore.vue:164]: Design is different . I have used swiper for above lg devices.

File: vue-frontend/src/components/contributions/GithubContribution.vue [hunk 4]
```diff
@@ -123,25 +123,27 @@
...
               :src="contribution.deskImage[1]"
               :srcset="`${contribution.deskImage[0]} 400w, ${contribution.deskImage[1]} 800w`"
-              class="tw-absolute tw-w-full tw-h-full tw-object-cover tw-rounded-[20px] card-image card-image-1"
+              class="tw-absolute tw-w-full lg:tw-h-full tw-object-cover tw-object-contain tw-rounded-[20px] card-image card-image-1"
               alt="contribution-image"
             />
           </div>
           <div
-            class="flip-card-back tw-absolute tw-w-full tw-h-full tw-rounded-[20px] tw-from-[#FF835B] tw-to-[#F2709C] tw-bg-gradient-[180deg] card-image back-card"
+            class="flip-card-back tw-absolute tw-w-full tw-h-[92%] lg:tw-h-full tw-rounded-[20px] tw-from-[#FF835B] tw-to-[#F2709C] tw-bg-gradient-[180deg] card-image back-card"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/contributions/GithubContribution.vue:146]: Fix following UI break

Screenshot from 2023-06-21 17-30-30

Reduce verticle space between cards in md devices
Screenshot from 2023-06-21 17-31-12

File: vue-frontend/src/components/contributions/DesignExplore.vue [hunk 5]
```diff
@@ -0,0 +1,340 @@
+<template>
+  <div class="xll:tw-container tw-my-20 md:tw-mb-40">
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/contributions/DesignExplore.vue:2]: Reduce bottom space

File: vue-frontend/src/components/contributions/Favourite.vue [hunk 6]
```diff
@@ -2,12 +2,12 @@
   <section class="tw-my-20 md:tw-mt-10">
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/contributions/Favourite.vue:2]: Increase top space on md devices, its too small

File: vue-frontend/src/pages/contributions.vue [hunk 7]
```diff
@@ -87,6 +92,7 @@ export default {
     AnimatedCreation,
     UnitTest,
```
Comment (cp-sumi-k) [inline vue-frontend/src/pages/contributions.vue:93]: Reduce size of images in md devices for unit test images

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**